lib/autoyast.rb

```diff
@@ -101,6 +100,12 @@ def profile
     builder.to_xml
   end
 
+  def outgoing_ip
+    output = Cheetah.run("ip", "route", "ls", stdout: :capture)
```

Comment:
At least online I saw an output of `ip route` that didn't show the default route in the first line so I am not sure if it will always work. That's why I have created the regex.